.github/workflows/ci_skipped.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI Skipped
+on:
+  pull_request:
+    branches: ["*"]
```

Comment:
Keep in mind this only works for branches without slashes (like "main") but fails when they do contain it (like with "feat/added-something")

Reply:
changed to a `**` here [8f84a9d](https://github.com/lambdaclass/lambda_ethereum_rust/pull/970/commits/8f84a9d7d66bf09619e0b84bce488a7116775da2)